### DISCUSSION TITLE
[Vue warn]: Failed to resolve component: picker

### DISCUSSION THREAD

**Discussion by holyrong** [NervJS/taro#11090] [Q & A]
版本：Taro3.4 NutUI3.1.14  typescript  vue3 
按需加载：
![image](https://user-images.githubusercontent.com/8318475/149123784-a91a6b3e-2e44-4764-a4c7-5626f25895e0.png)

vue页面代码：
`<nut-picker
                mode="multiSelector"
                :list-data="tenants"
                :default-value-index="currentIndex"
                @confirm="onChange"
              ></nut-picker>`

vscode编译正常，小程序控制台报以下错误：
` [Vue warn]: Failed to resolve component: picker`

![image](https://user-images.githubusercontent.com/8318475/149124199-18d19f3e-7064-4839-a77b-9383b7ff584f.png)

**Comment by sunjinwen** [comment]
我也碰到同样的问题，请问下解决这个问题了吗

**Comment by xuwu125** [comment]
请问大家是如何解决这个问题的？

**Comment by Chen-jj** [comment]
新建一个 issue，附上 demo 看看咯
